docs/routing.rst

```diff
@@ -73,6 +73,15 @@ in user code.
                 ...
 
 
+    .. method:: fetch(event)
+
+        Return a set of asynchronous callbacks registered
+        to the router that the *event* would be called on. The *event*
+        argument corresponds to :class:`gidgethub.sansio.Event`.
+
+        .. versionadded:: 4.1.2
```

Comment:
4.1.1 is the last released version in PyPI, but we already have 4.2.0 in changelog: https://gidgethub.readthedocs.io/en/latest/changelog.html#id1

Perhaps this change should be included in v.4.2.0.
```suggestion
        .. versionadded:: 4.2.0
```